src/Log/LogService.php

```diff
@@ -353,19 +357,17 @@ protected function createLogLevelFilter($logLevel)
      */
     public function getLog($logName = null)
     {
-        if ($logName = null) {
-            return getDefaultLog();
-        }
-        if ($logName == 'default') {
+        if ($logName = null || $logName == 'default') {
```

Comment:
This line sets $logName to null, always.

Reply:
Fixing it.